libs/core/src/lib/tabs/tab-item/tab-item.directive.ts

```diff
@@ -48,7 +48,7 @@ export class TabItemDirective implements CssClassBuilder, OnChanges, OnInit {
     linkItem: TabLinkDirective;
 
     /** @hidden */
-    constructor(private _elementRef: ElementRef) {}
+    constructor(public readonly elementRef: ElementRef) {}
```

Comment:
Do we want to keep all these files in sync? I see above we refactored these from construction to properties used `inject()` function and why not in these classes?

Reply:
The main task is about platform table components. These are the side effects and can be updated separately.